[2024-10-08T11:31:27.108+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-08T11:31:27.119+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trigger_dag.wait_for_run_file manual__2024-10-08T11:31:25.205749+00:00 [queued]>
[2024-10-08T11:31:27.123+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trigger_dag.wait_for_run_file manual__2024-10-08T11:31:25.205749+00:00 [queued]>
[2024-10-08T11:31:27.123+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-08T11:31:27.131+0000] {taskinstance.py:2888} INFO - Executing <Task(FileSensor): wait_for_run_file> on 2024-10-08 11:31:25.205749+00:00
[2024-10-08T11:31:27.135+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=521) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T11:31:27.137+0000] {standard_task_runner.py:72} INFO - Started process 523 to run task
[2024-10-08T11:31:27.137+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_dag', 'wait_for_run_file', 'manual__2024-10-08T11:31:25.205749+00:00', '--job-id', '247', '--raw', '--subdir', 'DAGS_FOLDER/trigger_dag.py', '--cfg-path', '/tmp/tmp9ku_biug']
[2024-10-08T11:31:27.138+0000] {standard_task_runner.py:105} INFO - Job 247: Subtask wait_for_run_file
[2024-10-08T11:31:27.176+0000] {task_command.py:467} INFO - Running <TaskInstance: trigger_dag.wait_for_run_file manual__2024-10-08T11:31:25.205749+00:00 [running]> on host 99c9ec34a420
[2024-10-08T11:31:27.236+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='trigger_dag' AIRFLOW_CTX_TASK_ID='wait_for_run_file' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T11:31:25.205749+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T11:31:25.205749+00:00'
[2024-10-08T11:31:27.238+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-08T11:31:27.238+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-08T11:31:27.238+0000] {logging_mixin.py:190} INFO - Current task name:wait_for_run_file state:running start_date:2024-10-08 11:31:27.119420+00:00
[2024-10-08T11:31:27.238+0000] {logging_mixin.py:190} INFO - Dag name:trigger_dag and current dag run status:running
[2024-10-08T11:31:27.238+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-08T11:31:27.239+0000] {baseoperator.py:405} WARNING - FileSensor.execute cannot be called outside TaskInstance!
[2024-10-08T11:31:27.245+0000] {base.py:84} INFO - Retrieving connection 'fs_default'
[2024-10-08T11:31:27.246+0000] {filesystem.py:109} INFO - Poking for file /opt/airflow/data/small_text.txt
[2024-10-08T11:32:27.251+0000] {filesystem.py:109} INFO - Poking for file /opt/airflow/data/small_text.txt
[2024-10-08T11:33:27.263+0000] {filesystem.py:109} INFO - Poking for file /opt/airflow/data/small_text.txt
[2024-10-08T11:33:27.268+0000] {filesystem.py:113} INFO - Found File /opt/airflow/data/small_text.txt last modified: 20241008103711
[2024-10-08T11:33:27.269+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2024-10-08T11:33:27.270+0000] {filesystem.py:109} INFO - Poking for file /opt/airflow/data/small_text.txt
[2024-10-08T11:33:27.271+0000] {filesystem.py:113} INFO - Found File /opt/airflow/data/small_text.txt last modified: 20241008103711
[2024-10-08T11:33:27.300+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-08T11:33:27.301+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=trigger_dag, task_id=wait_for_run_file, run_id=manual__2024-10-08T11:31:25.205749+00:00, execution_date=20241008T113125, start_date=20241008T113127, end_date=20241008T113327
[2024-10-08T11:33:27.316+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-08T11:33:27.316+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-08T11:33:27.317+0000] {logging_mixin.py:190} INFO - Dag name:trigger_dag queued_at:2024-10-08 11:31:25.216577+00:00
[2024-10-08T11:33:27.317+0000] {logging_mixin.py:190} INFO - Task hostname:99c9ec34a420 operator:FileSensor
[2024-10-08T11:33:27.358+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-08T11:33:27.381+0000] {dagrun.py:977} ERROR - Failed to get task for ti <TaskInstance: trigger_dag.process_results.small_text.txt.txt manual__2024-10-08T11:31:25.205749+00:00 [None]>. Marking it as removed.
[2024-10-08T11:33:27.399+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-08T11:33:27.401+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
